packages/tpetra/core/src/Tpetra_Details_crsUtils.hpp

```diff
@@ -375,6 +375,10 @@ pad_crs_arrays(
   indices_wdv = Indices(indices_new);
   values_wdv = Values(values_new);
 
+  // drop device views before getting host in verbose block
+  indices_new = decltype(indices_new)();
+  values_new = decltype(values_new)();
```

Comment:
You could just wrap the usage of the two views into its own code block. But I guess this is also fine.